### PR TITLE
Adds support for dynamically populating vendor apps

### PR DESCRIPTION
**Pull request by GrantMatejka**
I need a formal CR on this

### CODE REVIEW

File: api/server.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable  */
```
Comment (nauticalmonkey) [inline api/server.js:1]: Which linter rule are you trying to bypass i can fix the linebeaks/did already

**Review by nauticalmonkey — APPROVED** [review]

**Comment by nauticalmonkey** [comment]
I can fix the linter rules, lets discuss what we want to do with it

**Comment by GrantMatejka** [comment]
Sounds good, we will address it in a future pull

File: api/server.js [hunk 2]
```diff
@@ -13,58 +14,71 @@ process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
...
-            );
-         } else {
-            res.status(401).send();
-         }
-      })
-      .catch(function (error) {
-         console.log(error);
-         res.status(500).send();
-      });
+  db.oneOrNone('SELECT * FROM "Users" WHERE "Email" = $1 AND "Password" = $2', [req.body.email, req.body.hashedPass])
```
Comment (meinfr13nd) [inline api/server.js:21]: Shouldn't these errors have more meaningful messages? That would make them more user-friendly to non-technical users

File: apps/grant/src/index.js [hunk 3]
```diff
@@ -1,10 +1,10 @@
+/* eslint-disable  */
+
 const express = require('express');
 const axios = require('axios');
 const app = express();
 
-const cloudhaven = 'http://127.0.0.1:5000/api';
-// reference
-// https://blog.logrocket.com/how-to-make-http-requests-like-a-pro-with-axios/
+const chURL = 'http://127.0.0.1:5000/api';
```
Comment (meinfr13nd) [inline apps/grant/src/index.js]: This variable name isn't very meaningful (and it doesn't follow standard camel case formatting)

File: src/biz/APIParser.js [hunk 4]
```diff
@@ -16,7 +16,7 @@ export function parseJson(apiReturn) {
     if (comp === 'form') {
       return formGenerator(components.form);
     } else {
-      console.log('api error');
+      console.error('api error');
```
Comment (meinfr13nd) [inline src/biz/APIParser.js]: This would be a great opportunity to make this error message more meaningful

File: api/server.js (hunk 2 above)
Comment (GrantMatejka) [reply api/server.js:21]: Sounds great 🌽

File: apps/grant/src/index.js (hunk 3 above)
Comment (GrantMatejka) [reply apps/grant/src/index.js]: Thanks for the catch, should be fixed now

File: src/biz/APIParser.js (hunk 4 above)
Comment (GrantMatejka) [reply src/biz/APIParser.js]: 🌽

**Review by meinfr13nd — APPROVED** [review]
Love the changes! Look great!! 💯 🥇

**Comment by GrantMatejka** [comment]
Merging!

**Pull request merged**